packages/react-components/src/lib/context/AgoricContext.ts

```diff
@@ -14,9 +17,13 @@ export type PursesJSONState<T extends AssetKind> = {
   /** The petname for this purse */
   pursePetname: string;
   /** The brand's displayInfo */
-  displayInfo: unknown;
+  // XXX Copied from @agoric/ertp/src/types-ambient.js
```

Comment:
_question_

Just curious, what does `XXX` stand for here?

Reply:
To me it just signals a code smell that isn't really actionable enough for a `TODO`. Ideally the types would be exported from agoric-sdk as a module instead of being ambient. I'm not sure if there's an issue to track this exactly ([my search](https://github.com/search?q=repo%3AAgoric%2Fagoric-sdk+ambient+types&type=issues)).